### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by goderbauer**

### CODE REVIEW

File: engine/README.md [hunk 1]
```diff
@@ -6,5 +6,7 @@ Flutter engine uses `gclient` to manage dependencies.
 
 If you've already cloned the flutter repository:
 
-1. Copy one of the `engine/scripts/*.gclient` to the root folder as `.gclient`
+1. Copy one of the `engine/scripts/*.gclient` to the root folder as `.gclient`:
+    1. Googlers: copy `rbe.gclient` to enable faster builds with RBE
```
Comment (matanlurey) [inline engine/README.md:10]: I would also link to <https://github.com/flutter/flutter/blob/master/engine/src/flutter/docs/rbe/rbe.md>.

Comment (jtmcdole) [reply engine/README.md:10]: googlers could also replace the url with git@github link?

Comment (goderbauer) [reply engine/README.md]: Added the link to RBE.

**Review by jtmcdole — APPROVED** [review]
lgtm

**Pull request merged**